Services/Certificate/PRIVACY.md

```diff
@@ -0,0 +1,97 @@
+# Certificate Service Privacy
+
+This documentation does not warrant completeness or correctness. Please report any
+missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de)
+or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+
+- The `Certificate Service` can be enabled in the global administration. If enabled, certificate templates can be
```

Comment:
An account with "Edit Settings" permissions Administration > Achiervements > Certificates > Settings can enable the Certificate Service globally.